markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -249,24 +249,32 @@ library PerpsAccount {
         GlobalPerpsMarket.load().updateCollateralAmount(collateralId, amountDelta);
     }
 
-    function payDebt(Data storage self, uint256 amount) internal {
+    function payDebt(Data storage self, uint256 amount) internal returns (uint256 debtPaid) {
         if (self.debt == 0) {
             revert NonexistentDebt(self.id);
         }
 
+        /*
+            1. if the debt is less than the amount, set debt to 0 and only deposit debt amount
+            2. if the debt is more than the amount, subtract the amount from the debt
+            3. excess amount is ignored
+        */
+
         PerpsMarketFactory.Data storage perpsMarketFactory = PerpsMarketFactory.load();
+
+        if ((self.debt.toInt() - amount.toInt()) > 0) {
```

Comment:
Why don't just `if (self.debt > amount) { }` ? That way reading is direct with the comment above.

Reply:
lol yea much better